updater/versions/update-1.py

```diff
@@ -72,13 +71,22 @@ def run():
     rm_rf("/etc/cron.daily/remove_upl_reports")
     rm_rf("/etc/cron.daily/run_ooniprobe_deck")
     rm_rf("/etc/cron.daily/upload_reports")
-    rm_rf("/etc/cron.daily/remove_inc_reports")
-    rm_rf("/etc/cron.daily/update_ooniprobe_deck")
+
+    # Delete all the weekly crons
+    rm_rf("/etc/cron.weekly/remove_inc_reports")
+    rm_rf("/etc/cron.weekly/update_ooniprobe_deck")
+
+    # Remove unneeded cronjobs
+    rm_rf("/etc/cron.daily/apt")
+    rm_rf("/etc/cron.daily/dpkg")
+    rm_rf("/etc/cron.daily/man-db")
+    rm_rf("/etc/cron.daily/tor")
 
     # XXX this is still present in the lepidopter v2 branch.
     rm_rf("/etc/cron.daily/update_ooniprobe")
 
-    rm_rf("/etc/ooniprobe")
```

Comment:
Is there anything else we need inside of `/etc/ooniprobe`. It seems to me like nothing except `ooniprobe.conf` would be used, but even that, since the cronjobs are getting dropped, should no longer be required. Am I mistaken?